base/wagtail_hooks.py

```diff
@@ -0,0 +1,25 @@
+from wagtail.snippets.models import register_snippet
```

Comment:
Would be helpful to demonstrate a wagtail_hook, such as when the page is published. Even if it's just a terminal message to display how it works.

Reply:
I’d love to. Currently our goal is to only implement things that are on the bakerydemo, so we have two hooks to choose from: https://github.com/wagtail/bakerydemo/blob/2edb8745895916f6090662f13f7e35600fb36727/bakerydemo/base/wagtail_hooks.py#L1-L42

(or add a new hook to the bakerydemo)